### ISSUE TITLE
changing the font globally

### ISSUE THREAD

**Issue by ghost** [ys1998/academic-portfolio#1]
Hi, I wanted to ask if you could walk me through how to change the font for this theme?

In main.scss I changed this variable, but it doesn't seem to have an effect:

$base-font-family: 'Linux Biolinum', sans-serif;

**Comment by d-jiao** [comment]
Yes, this has been how I have set up the global configurations for fonts, color maps, etc. Please kindly make sure the designated font is legitimate.
